rls/src/main/java/io/grpc/rls/RlsProtoConverters.java

```diff
@@ -153,7 +153,7 @@ protected RouteLookupConfig doForward(Map<String, ?> json) {
         maxAge = MAX_AGE_NANOS;
       }
       if (staleAge == null) {
-        staleAge = MAX_AGE_NANOS;
+        staleAge = maxAge >= MINUTES.toNanos(2) ? maxAge - MINUTES.toNanos(1) : maxAge;
```

Comment:
this makes sense to me for a quick mitigation. but fyi 
>   // This value should be less than max_age by at least the length of a
  // typical RTT to the Route Lookup Service to fully mask the RTT latency.

from [proto](https://github.com/grpc/grpc-java/blob/master/rls/src/main/proto/grpc/lookup/v1/rls_config.proto#L193C5-L193C5)

worth commenting why grpc choose stale age like in the proto.